src/main/java/in/twizmwaz/cardinal/module/modules/teamPicker/TeamPicker.java

```diff
@@ -52,12 +55,20 @@ public Inventory getTeamPicker(Player player) {
                 totalPlayers += team.size();
             }
         }
```

Comment:
When you're doing the below changes, they only show or go away when the player's inventory is reset. If I un-livestreamer someone, they still keep the "livestreamer inventory".